kolibri/core/assets/src/views/core-base.vue

```diff
@@ -104,11 +106,13 @@
         return this.navShown ? this.headerHeight * 4 : 0;
       },
       appBarStyle() {
-        return this.mobile ? {} : { paddingLeft: `${this.navWidth + PADDING}px` };
+        const posKey = Vue.bidiDirection === 'rtl' ? 'Right' : 'Left';
+        return this.mobile ? {} : { ['padding' + posKey]: `${this.navWidth + PADDING}px` };
       },
       contentStyle() {
         const style = { top: `${this.headerHeight}px` };
-        style.left = this.mobile ? 0 : `${this.navWidth}px`;
+        const posKey = Vue.bidiDirection === 'rtl' ? 'right' : 'left';
+        style[posKey] = this.mobile ? 0 : `${this.navWidth}px`;
```

Comment:
Yes, can definitely do some further design on this in my larger i18n refactor.

I am a little reticent to bundle in rtlcss utilities at runtime though, as it invites mindless flipping in all cases. And the number of times this happens in Javascript is still too small to justify bundling in 9Kb of utilities.